argo_workflows/assets/dashboards/argo_workflows.json

```diff
@@ -0,0 +1,756 @@
+{
+    "author_name": "Datadog",
+    "description": "[[suggested_dashboards]]",
```

Comment:
We'll need a better description 😆

Reply:
huh? I totally didn't see this in the UI... 🔍 now...